build.sbt

```
@@ -60,7 +59,7 @@ lazy val dockerSettings = Seq(
   dockerUpdateLatest := updateLatest.value,
   dockerCommands ++= Seq(
     Cmd("USER", "root"),
-    Cmd("RUN", "apk update && apk add bash")
+    Cmd("RUN", "apk update && apk add bash && apk add eudev")
```

```suggestion
    Cmd("RUN", "apk update && apk add bash eudev")
```

Can just list the packages after add.

Reference: https://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management#Add_a_Package